docs/decisions/0003-initial-api-specification.rst

```diff
@@ -93,3 +93,145 @@ Permissions
 enterprise_learner
   Should only create the transaction if the requesting user has implicit (JWT) or explicit (DB-defined)
   ``enterprise_learner`` role assigned.
+
+
+CRUD operations on SubsidyAccessPolicy
+======================================
+
+GET **/api/v1/admin/policy/**
+
+List all policies linked to a group.
+
+Inputs
+------
+
+- ``group_uuid`` (Query Param, required): The uuid of the customer. For now it will be an enterprise customer uuid.
```

Comment:
i am wondering if we should use `enterprise_customer_uuid` here rather than group. group is going to have a very specific meaning in the future - a way to group learners within an enterprise. we will still need to be able to query based on enterprise customer to find all policies regardless of the (future) groups associated with the policies.

Reply:
I would like to +1 this. I think that the language here needs to be very precise.

Reply:
Created https://2u-internal.atlassian.net/browse/ENT-6902 to take care of this.